docs/docs/cmd/spo/list/list-roleinheritance-break.md

```diff
@@ -11,13 +11,16 @@ m365 spo list roleinheritance break [options]
 ## Options
 
 `-u, --webUrl <webUrl>`
-: URL of the site where the list to retrieve is located
+: URL of the site.
 
 `-i, --listId [listId]`
-: ID of the list to retrieve information for. Specify either id or title but not both
+: ID of the list. Specify either `listTitle`, `listId` or `listUrl`.
 
 `-t, --listTitle [listTitle]`
-: Title of the list to retrieve information for. Specify either id or title but not both
+: Title of the list. Specify either `listTitle`, `listId` or `listUrl`.
+
+`--listUrl [listUrl]`
+: Relative URL of the list. Specify either `listTitle`, `listId` or `listUrl`.
```

Comment:
Are we expecting server- or site-relative URL? Let's disambiguate it. If we allow both, let's make it explicit.

Reply:
We allow both. I've changed this.